source/validate.cpp

```diff
@@ -66,6 +68,14 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInsts,
   return SPV_SUCCESS;
 }
 
+// Returns true if the input is a known extension.
+// TODO: The list of recognized extensions should be populated atuomatically.
```

Comment:
But from what.   What does a maintainer do when a new extension comes along?  Do they add the string here first, and then implement the extensions checks?  Should say so.